lib/shopify_import_massager.ex

```diff
@@ -1,6 +1,6 @@
 defmodule ShopifyImportMassager do
   @moduledoc """
-  Massages csv import files, specifically for Maude.
+  Massages csv import files, specifically for Minnowswim.
   """
 
   @order_number_regex ~r|^#(\d+)$|
```

Comment:
Note that `convert_files` (the function I can't comment on because I haven't changed it) has a list of parameters and is a very long function. Hard to maintain.